packages/venia-concept/src/components/Checkout/form.js

```diff
@@ -0,0 +1,46 @@
+import { Component, createElement } from 'react';
+import { func, shape, string } from 'prop-types';
+
+import classify from 'src/classify';
+import Section from './section';
+import SubmitButton from './submitButton';
+import defaultClasses from './form.css';
+
+class Form extends Component {
+    static propTypes = {
+        classes: shape({
+            footer: string,
+            root: string,
+            sections: string
+        }),
+        status: string,
+        submitOrder: func
+    };
+
+    render() {
+        const { classes, status, submitOrder } = this.props;
+        const today = new Date().toDateString();
+
+        return (
+            <div className={classes.root}>
+                <div className={classes.body}>
+                    <Section label="Ship To">
+                        <span>Add Shipping Information</span>
```

Comment:
We always want to add TODO comments for unimplemented sections like this and below.

Reply:
True. I'll add some. 👍